<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>informationleft</class>
 <widget class="QDialog" name="informationleft">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>620</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>260</width>
    <height>620</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>采集组数:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_groupcnt">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>300</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1/80</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar_grouppercent">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>225</width>
     <height>23</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QProgressBar::chunk {
   background-color: #2077d4;
   width: 20px;
}

QProgressBar {
   border: 1px solid grey;
   border-radius: 0px;
   text-align: center;
}</string>
   </property>
   <property name="value">
    <number>10</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>当前角度：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_currentAngle">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>数据文件：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_filename1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>chx</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_filename2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>440</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>chx</string>
   </property>
  </widget>
  <widget class="QwtDial" name="Dial">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>50</y>
     <width>211</width>
     <height>211</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="lowerBound">
    <double>0.000000000000000</double>
   </property>
   <property name="upperBound">
    <double>360.000000000000000</double>
   </property>
   <property name="scaleMaxMajor">
    <number>12</number>
   </property>
   <property name="scaleMaxMinor">
    <number>3</number>
   </property>
   <property name="scaleStepSize">
    <double>30.000000000000000</double>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="frameShadow">
    <enum>QwtDial::Plain</enum>
   </property>
   <property name="origin">
    <double>270.000000000000000</double>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtDial</class>
   <extends>QWidget</extends>
   <header>qwt_dial.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
